src/requestHandlers/populate.ts

```diff
@@ -0,0 +1,87 @@
+import { sanitizeResponse } from './utils';
+import { Request, Response } from 'express';
+import * as PanelSetService from '../services/panelSetService';
+import { Sequelize } from 'sequelize';
+import { sequelize } from '../database';
+import * as UserService from '../services/userService';
+import { _createHookController } from './hook';
+import * as PanelService from '../services/panelService';
+import { IPanelSet, IPanel, IUser } from '../models';
+
+// test methods so I don't have to run all of these queries every time I want to test something
+const populate = async (request: Request, res: Response) => {
+    const response = await _populate(sequelize)();
+    return sanitizeResponse(response, res, '');
+};
```

Comment:
When this is ran, the following message pops up: `"Panel set is a trunk"`. I understand that this is because of the thing hook validation, but I would refactor this in this pr as the query is unusable otherwise.